_about/notes/0.8.md

```diff
@@ -8,4 +8,55 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
+
+- **Revamped Traffic Management Model**. We're finally ready to take the wraps off our new traffic
+management configuration model. The new model adds many new features and addresses usability issues
+with the prior model. There is a conversion tool built into `istioctl` to help migrate your config from
+the old model. [Learn more about the new traffic management model]({{home}}/docs/tasks/traffic-management-v1alpha3/).
+
+- **Envoy V2**. Users can choose to inject Envoy V2 as the sidecar. In this mode, Pilot uses the V2 Envoy API to program the proxies
+in the mesh, which increases effective scalability and should eliminate spurious 404 errors. [TBD: docs on how to control this?]
+
+- **Constrain Inbound Ports**. Restrict the inbound ports in a pod to the ones needed by the apps running inside that pod.
```

Comment:
Call it Security fixes: Inbound ports on a pod are restricted to those declared by the app in the YAML